### DISCUSSION TITLE
While creating certificate the name and designation of the signatory are not getting saved and reflected on the certificate later

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#460] [General Discussion]
**While creating a certificate or template using the layout,** The title and certificate name is reflected on the certificate template, and the logos are also reflected but **the signatory name and designation are not reflected.** 
**How can we solve this issue??**
Please guide us on how to do it.
We are using Sunbird Saas version 4.6.
![image](https://user-images.githubusercontent.com/109504832/231850325-c1a9322c-8b5c-4a2c-90cd-2fc05327e036.png)
[@vinukumar-vs](https://github.com/vinukumar-vs) [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Comment by chitranshu-keshav** [comment]
@vinukumar-vs @rajeevsathish Can you please get back on the above

**Reply by chitranshu-keshav** [reply]
@vinukumar-vs @rajeevsathish  can you please get back on this or loop in the right person who can resolve the above issue

**Reply by rajeevsathish** [reply]
@sachinb80 this looks like a templet issue please create a new templet and try the same this should work fine

cc: @chitranshu-keshav @swayangjit @vinukumar-vs

**Reply by sachinb80** [reply]
@rajeevsathish  Tried creating a new template but still not able to update or change the signatory name and designation.
**Everything is getting updated but not the highlighted part in the screenshot below**
![image](https://user-images.githubusercontent.com/109504832/233276933-ea23d716-3afb-46bf-b627-bb56537cbc64.png)

**Reply by rajeevsathish** [reply]
@sachinb80 post changing try to refresh or update the image once and then you will be able to see the changes back

**Reply by sachinb80** [reply]
@rajeevsathish Yes I did try that but still, it is not updating. After refreshing the title, name, and logos are getting updated but not the signatory name and designation.

**Reply by rajeevsathish** [reply]
@sachinb80 in sunbird release-5.1.0 and also in release-5.2.0 we are not seeing this issue so can you please update your build to latest and try once.
cc: @chitranshu-keshav @vinukumar-vs @Krishnaj20

**Reply by chitranshu-keshav** [reply]
@sachinb80 is your issue sorted out by updating the build? If not then can you please get back with the details

**Reply by Krishnaj20** [reply]
Locking this conversation as we haven't heard back from @sachinb80 .
